packages/camera/camera_windows/lib/camera_windows.dart

```diff
@@ -0,0 +1,454 @@
+      final List<Map<dynamic, dynamic>>? cameras = await _channel
+          .invokeListMethod<Map<dynamic, dynamic>>('availableCameras');
+
+      if (cameras == null) {
+        return <CameraDescription>[];
+      }
+
+      return cameras.map((Map<dynamic, dynamic> camera) {
+        return CameraDescription(
+          name: camera['name'] as String,
+          lensDirection:
+              parseCameraLensDirection(camera['lensFacing'] as String),
+          sensorOrientation: camera['sensorOrientation'] as int,
+        );
+      }).toList();
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+  }
+
+  @override
+  Future<int> createCamera(
+    CameraDescription cameraDescription,
+    ResolutionPreset? resolutionPreset, {
+    bool enableAudio = false,
+  }) async {
+    try {
+      final Map<String, dynamic>? reply = await _channel
+          .invokeMapMethod<String, dynamic>('create', <String, dynamic>{
+        'cameraName': cameraDescription.name,
+        'resolutionPreset': resolutionPreset != null
+            ? _serializeResolutionPreset(resolutionPreset)
+            : null,
+        'enableAudio': enableAudio,
+      });
+
+      if (reply == null) {
+        throw CameraException('System', 'Cannot create camera');
+      }
+
+      return reply['cameraId']! as int;
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+  }
+
+  @override
+  Future<void> initializeCamera(
+    int cameraId, {
+    ImageFormatGroup imageFormatGroup = ImageFormatGroup.unknown,
+  }) async {
+    final int requestedCameraId = cameraId;
+
+    //Create channel for camera events
+    _channels.putIfAbsent(requestedCameraId, () {
+      final MethodChannel channel =
+          MethodChannel('flutter.io/cameraPlugin/camera$requestedCameraId');
+      channel.setMethodCallHandler(
+        (MethodCall call) => handleCameraMethodCall(call, requestedCameraId),
+      );
+      return channel;
+    });
+
+    final Map<String, double>? reply;
+    try {
+      reply = await _channel.invokeMapMethod<String, double>(
+        'initialize',
+        <String, dynamic>{
+          'cameraId': requestedCameraId,
+        },
+      );
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+
+    if (reply != null &&
+        reply.containsKey('previewWidth') &&
+        reply.containsKey('previewHeight')) {
+      cameraEventStreamController.add(
+        CameraInitializedEvent(
+          requestedCameraId,
+          reply['previewWidth']!,
+          reply['previewHeight']!,
+          ExposureMode.auto,
+          false,
+          FocusMode.auto,
+          false,
+        ),
+      );
+    } else {
+      throw CameraException(
+        'INITIALIZATION_FAILED',
+        'The platform "$defaultTargetPlatform" did not return valid data when reporting success. The platform should always return a valid data or report an error.',
```

Comment:
Although in this case, it sounds like it's describing an implementation bug in the native code; providing an exception with an error message isn't useful in that case since the developer who would get it can't act on this.

Just use `!` (which will throw an exception if there's a bug in the plugin).

This applies below as well.

Reply:
This should be fixed here as well: [method_channel_camera.dart](https://github.com/flutter/plugins/blob/main/packages/camera/camera_platform_interface/lib/src/method_channel/method_channel_camera.dart#L209-L212)

FIXED